CairoMakie/src/primitives.jl

```diff
@@ -204,10 +204,13 @@ end
 # an array of markers is converted to string by itself, which is inconvenient for the iteration logic
 _marker_convert(markers::AbstractArray) = map(m -> convert_attribute(m, key"marker"(), key"scatter"()), markers)
 _marker_convert(marker) = convert_attribute(marker, key"marker"(), key"scatter"())
+# image arrays need to be converted as a whole
+_marker_convert(marker::AbstractArray{<:Colorant}) = convert_attribute(marker, key"marker"(), key"scatter"())
```

Comment:
Maybe make this AbstractMatrix?  Might be too general like this.

Reply:
Indeed, thanks for spotting.